### PR TITLE
Fix autoranging on Plots.Segment and Plots.Rectangle (alternate take)

### LINKED ISSUE #2463: Autoranging does not work correctly for Plots.Rectangle and Plots.Segment

### PR DESCRIPTION
**Pull request by jtlan**
Fixes #2463

### CODE REVIEW

File: src/plots/plot.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ export class Plot extends Component {
   private _updateExtentsForKey(key: string, bindings: d3.Map<Plots.AccessorScaleBinding<any, any>>,
       extents: d3.Map<any[]>, filter: Accessor<boolean>) {
     var accScaleBinding = bindings.get(key);
-    if (accScaleBinding.accessor == null) { return; }
+    if (accScaleBinding == null || accScaleBinding.accessor == null) { return; }
```
Comment (rcchen) [inline src/plots/plot.ts]: Nice.

**Comment by crmorford** [comment]
https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0CAcQjRxqFQoTCPGMp-rl4sYCFQs_PgodJqUOPA&url=http%3A%2F%2Fwww.livememe.com%2Fdzkynfl&ei=cEmpVdC6GsmT-wGo_IeoCA&bvm=bv.98197061,d.cWw&psig=AFQjCNGHUdq5h-ZO4MXt_A3h6M1K6S2Fig&ust=1437244150274452

**Comment by rcchen** [comment]
@crmorford just realized that there was no description attached to the PR, I added the issue that it fixes and the context is there. Sorry about that!

**Comment by crmorford** [comment]
Thanks :)

**Comment by jtlan** [comment]
> context

My bad :/

**Pull request merged**